src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/InProcess/EditorInProcess_Text.cs

```diff
@@ -160,10 +160,4 @@ public async Task WaitForActiveWindowByFileAsync(string fileName, CancellationTo
             TimeSpan.FromMilliseconds(50),
             cancellationToken);
     }
-
-    public async Task WaitForProjectReadyAsync(CancellationToken cancellationToken)
```

Comment:
This wasn't called, and does the exact same work as `WaitForProjectSystem` anyway